api/app/serializers/spree/v2/storefront/shipment_serializer.rb

```diff
@@ -0,0 +1,16 @@
+module Spree
+  module V2
+    module Storefront
+      class ShipmentSerializer < BaseSerializer
+        set_type :shipment
+
+        attributes :tracking, :number, :cost, :shipped_at, :state, :created_at, :updated_at,
+                   :adjustment_total, :additional_tax_total, :promo_total, :included_tax_total,
+                   :pre_tax_amount, :taxable:adjustment_total, :non_taxable_adjustment_total
+
+        belongs_to :stock_location
+        belongs_to :address
```

Comment:
@VygovskySergey this association to address is still WIP - we should not expose it for now